### PR TITLE
Fix to Space SIGINT XML

### PR DESCRIPTION
**Pull request by abouffard**

### CODE REVIEW

File: instance/jmsml_D_SIGINT_Space.xml [hunk 1]
```diff
@@ -284,7 +284,7 @@
     </LegacySymbol>
     <LegacySymbol ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_SYM" Label="I*PPSRU-----***" EntityID="SIGNAL_INTERCEPT" EntityTypeID="RADAR" ModifierOneID="UNKNOWN_MOD">
       <LegacyFunctionCode Name="2525C">SRU---</LegacyFunctionCode>
-	    <LegacyEntity ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_LE" Label="I-P-SRU---.svg">
+	    <LegacyEntity ID="SIGNAL_INTERCEPT_RADAR_UNKNOWN_MOD_LE" Label="SIGINT.SPC.SIGINC.RAD.UNK" Graphic="I-P-SRU---.svg">
```
Comment (csmoore) [inline instance/jmsml_D_SIGINT_Space.xml]: @abouffard @Dbarnes1 - this one line cause **all of Appendix D** not to draw? How is that possible? Is this PR the complete fix?

File: samples/legacy_support/All_ID_Mapping_Original.csv [hunk 2]
```diff
@@ -2579,7 +2579,7 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 2577,SIGINT.SPC.SIGINC.RAD.MFN,I-P-SRM---,I-P-SRM---,,,,,Point,,,
 2578,SIGINT.SPC.SIGINC.RAD.TGTAQ,I-P-SRT---,I-P-SRT---,,,,,Point,,,
 2579,SIGINT.SPC.SIGINC.RAD.SPC,I-P-SRS---,I-P-SRS---,,,,,Point,,,
-2580,I-P-SRU---.svg,I-P-SRU---,I-P-SRU---,,,,,Point,,,
+2580,SIGINT.SPC.SIGINC.RAD.UNK,I-P-SRU---,I-P-SRU---,,,,,Point,,,
```
Comment (csmoore) [inline samples/legacy_support/All_ID_Mapping_Original.csv]: I didn't realize these were using the hierarchy codes (line above) vs. human-readable names (see line below) - I opened a question about this here: https://github.com/Esri/joint-military-symbology-xml/issues/292

File: instance/jmsml_D_SIGINT_Space.xml (hunk 1 above)
Comment (abouffard) [inline instance/jmsml_D_SIGINT_Space.xml]: This one line caused an exception error in the export code which caused the remainder of appendix D not to be exported to CSV.  This PR fixed the line and also provided the updated CSV with it and the remainder of Appendix D inside.  Just waiting on @Dbarnes1 to verify.

Comment (csmoore) [inline instance/jmsml_D_SIGINT_Space.xml]: So is there also a code change (like an exception handler added) associated with this fix?

Comment (abouffard) [inline instance/jmsml_D_SIGINT_Space.xml]: I am working on one, yes.

**Pull request merged**